docs/sql/grammar.md

```diff
@@ -321,6 +322,35 @@ the column names are *not* used to reorder columns.
 In `orderItem`, if expression is a positive integer n, it denotes the
 nth item in the `SELECT` clause.
 
+## Creating indexes
+
+Feldera supports the `CREATE INDEX` SQL statement with the following
+syntax:
+
+```
+CREATE INDEX identifier ON identifier(columnOrList);
+```
+
+For the purpose of incremental view maintenance Feldera automatically
+creates all required indexes by analyzing the query defining the view.
+The SQL `CREATE INDEX` statement is used for a different purpose in
+Feldera than in standard databases; the statement is used to specify
+*primary keys* for output views.  Consider the following SQL program
```

Comment:
We call them unique keys in other docs, so should probably use the same terminology here. Besides, there can be multiple indexes over the same view, so they are more like unique keys than primary keys